### PR TITLE
EZZY-game-BSC-token

### PR DESCRIPTION
**Pull request by uhomuho**
**NOTE**

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
3. The protocol is usually listed within 24 hours of merging the PR
4. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
5. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
6. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
7. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Name (to be shown on DefiLlama):
EZZY Game (EZY)

##### Twitter Link:
https://twitter.com/ezzy_game

##### List of audit links if any:
https://skynet.certik.com/projects/ezzy-game

##### Website Link:
https://ezzy.game/

##### Logo (High resolution, will be shown with rounded borders):
ezzy_logo

##### Current TVL:
100k$

##### Treasury Addresses (if the protocol has treasury)
0xb452bc9cead0b08c4ef99da0feb8e10ef6bb86bb

##### Chain:
BNB Chain

##### Short Description (to be shown on DefiLlama):
EZZY Game — simplest M2E (Move-to-Earn) and P2E (Play-to-Earn) game ever with Web3 elements. It allows you to generate and get EZY tokens by walking or jogging in the open air, or playing a simple game.

Get real reward for walking outside, playing the game or getting other achievements. All you need is your smartphone and game Sneakers. After downloading the app and registering, you can get trial Sneakers as a gift!

##### Token address and ticker if any:
0xb452bc9cead0b08c4ef99da0feb8e10ef6bb86bb

##### Category (full list at https://defillama.com/categories) *Please choose only one:
https://defillama.com/protocols/Gaming

##### Oracle used (Chainlink/Band/API3/TWAP or any other that you are using):
https://ezzy.game/api/outer/token/total

##### methodology (what is being counted as tvl, how is tvl being calculated):
Liqudity pool balance

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/EZZY-game-BSC-token exports TVL: 
        
 
 
    bsc                       60.04 k
    
    total                    60.04 k

**Comment by llamatester** [comment]
The adapter at projects/EZZY-game-BSC-token exports TVL: 
        
 
 
    bsc                       0
    bsc-staking               59.89 k
    staking                   59.89 k
    
    total                    0

File: projects/EZZY-game-BSC-token/index.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const { staking } = require("../helper/staking")
+
+const contract = "0xc056124727540Fc26C3AF44d2b6BA4FbF59C6909"
```
Comment (g1nt0ki) [inline projects/EZZY-game-BSC-token/index.js:3]: this contract belongs to pancake swap,c ant be included in your tvl or staking

Comment (uhomuho) [reply projects/EZZY-game-BSC-token/index.js:3]: But we have only liquidity pool, how can we count token's TVL with it? I tried to use pool2 adapter, but i get these error
image

Comment (g1nt0ki) [reply projects/EZZY-game-BSC-token/index.js:3]: okay, this pool doesnt belong to you.

Do your users stake these LPs in smart contracts controlled by your project?

https://docs.llama.fi/list-your-project/what-to-include-as-tvl

Comment (uhomuho) [reply projects/EZZY-game-BSC-token/index.js:3]: Our user doesn't stake LP tokens, but our team locked LP tokens for 3 years on umicrypt smartcontact. can we somehow to show on defilama info about liqidity pool or LP tokens locked?

Comment (g1nt0ki) [reply projects/EZZY-game-BSC-token/index.js:3]: sorry, we currently do not support that

**Pull request closed without merging**